xbmc/network/WebServer.h

```diff
@@ -19,9 +19,6 @@
  *
  */
 
-#include "system.h"
-
-#ifdef HAS_WEB_SERVER
```

Comment:
network/WebServer.h is included unconditionaly in xbmc/settings/SettingConditions.cpp, this will break if microhttpd.h is not found.

Reply:
fixed